### DISCUSSION TITLE
Upgrade to new release of WikiTraccs

### DISCUSSION THREAD

**Discussion by haroldvandekamp** [WikiTransformationProject/wikitraccs-releases#58] [Q&A] [answered]
What is the suggested approach to a update to a newer release of the WikiTraccs product?

In the documentation, I found how to install the product. However, I have not found information regarding the upgrade process and prevent me from having to start all over in the GUI from scratch. So, I want to make sure that when using the new version of the product, I don't have to set up the source URL and destination your URL configuration again.

**Comment by heinrich-ulbricht** [comment, marked as answer]
Good points. The update process is now covered in the [Installation and Update](https://www.wikitransformationproject.com/docs/reference/installation-and-update/) section of the documentation. Let me know if you need more info.

**Reply by haroldvandekamp** [reply]
That's exactly what I was looking for. Thanks @heinrich-ulbricht .
